client/src/modules/dashboard/Venues/components/VenueForm.tsx

```diff
@@ -61,7 +61,9 @@ const VenueForm: React.FC<VenueFormProps> = (props) => {
           isDisabled={loading}
```

Comment:
to get the same console.log, you need to change line 41 in this file to this


```ts
resolver: async (data, context, options) => {
      // you can debug your validation schema here
      console.log('formData', data)
      console.log('validation result', await resolver(data, context, options))
      return resolver(data, context, options)
    },
    ```
 
 I think the solution is to clear the error through zod or yub, that's why I mentioned I should have investigated it more, sorry for cutting corners

Reply:
The issue before https://github.com/freeCodeCamp/chapter/commit/2bce8e9f18384df111cd60e0b120d0a6fd9644b4 is:

At first we set the default value of `latitude` and `longitude` to `null`. As soon as we touch the inputs its value changes to an empty string `""` or `NaN` (if we're using `{ valueAsNumber: true }`). This seems to be the default behavior of RHF and can be seen in both Firefox and Chrome. Next our validators `IsLatitude()` and `IsLongitude()` gets passed this empty string `""` or `NaN` value for validations, and since both the validators treat `""` and `NaN` as invalid latitude or longitude coordinates we get the error mentioned [above](https://github.com/freeCodeCamp/chapter/pull/2310#issuecomment-1418704540)

Reply:
Thank you for providing more info, I understand the issue more now. 👍 💚

Reply:
> This should be similar to this discussion https://github.com/react-hook-form/react-hook-form/discussions/9875#discussioncomment-4877380, and if RTF doesn't currently clear the error, after the user has changed the value

I think in our case the error wasn't being cleared because our validators were being fed the wrong values and so the errors didn't go away.

In https://github.com/freeCodeCamp/chapter/commit/2bce8e9f18384df111cd60e0b120d0a6fd9644b4, the value for latitude and longitudes were only validated if they were non-empty or valid numbers. 

This allowed us to ignore the error. But then on testing with Firefox I found the issue where Firefox would allow arbitrary strings to be typed into number input.